Source/WebCore/css/CSSProperties.json

```diff
@@ -8465,7 +8465,9 @@
             "values": [
                 "auto",
                 "under",
-                "from-font"
+                "from-font",
+                "left",
+                "right"
```

Comment:
I think we should restrict these values behind a setting as long as they're not implemented.

You can use `"settings-flag"` like for `overflow: clip`

Reply:
I just finished implementing the left and right, so this should not be needed.